notebooks/olmocr-pdf-vlm/olmocr-pdf-vlm.ipynb

```diff
@@ -0,0 +1,594 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    device = device_widget(default="AUTO", exclude=["NPU"])
</pre><p>NPU can be used in VLMPipeline now</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2842/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Removed</p>